app/src/test/java/org/greenstand/android/TreeTracker/Extension.kt

```diff
@@ -0,0 +1,46 @@
+package org.greenstand.android.TreeTracker
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.Observer
+import java.util.concurrent.CountDownLatch
+import java.util.concurrent.TimeUnit
+import java.util.concurrent.TimeoutException
+
+fun <T> LiveData<T>.getOrAwaitValue(
```

Comment:
It doesn't look like anything in this file is used. Can we remove these methods?

Reply:
ok